docs/quickstarts/README.md

```diff
@@ -161,7 +165,7 @@ If you have created quick starts in the Hybrid Cloud Console before, start here:
 9. Open a pull request (PR) in the [GitHub repository](https://github.com/RedHatInsights/quickstarts) containing your update. Get your quick start reviewed by stakeholders and peer reviewers, as needed. Once you and any reviewers are happy with the content, work with engineering to add the content to the Hybrid Cloud Console.
```

Comment:
"work with engineering to add the content to the HCC." What exactly does this mean? I think we should spell this out for new users.

Reply:
Reworded a little - basically this meant 'continue to the next steps where you work together with engineering to merge your work'